Update install guide 5.0.1, add upgrading details (#1012)
sensu/sensu-docs: content/sensu-go/5.0/installation/upgrade.md

    parent: installation
---

- [Upgrading from 5.0.0 or later](#upgrading-to-the-latest-version-of-sensu-go-from-5-0-0-or-later)
- [Upgrading from 1.x or later](#upgrading-to-sensu-go-from-sensu-core-1-x)

## Upgrading to the latest version of Sensu Go from 5.0.0 or later

To upgrade to the latest version of Sensu Go from version 5.0.0 or later, first [install the latest packages][23].

Then restart the services.

{{< highlight shell >}}
# Restart the Sensu agent
sudo service sensu-agent restart

# Restart the Sensu backend
sudo service sensu-backend restart
{{< /highlight >}}

> For systems using `systemd`, you may need to run `sudo systemctl daemon-reload` after restarting the services.

Reviewer: ```suggestion
For systems using `systemd`, you should run `sudo systemctl daemon-reload` before restarting the services.
```

> You can use the `version` command to verify the version using the `sensu-agent`, `sensu-backend`, and `sensuctl` tools. For example: `sensu-backend version`.

Reviewer: ```suggestion
You can use the `version` command to determine the installed version using the `sensu-agent`, `sensu-backend`, and `sensuctl` tools. For example: `sensu-backend version`.
```

    parent: installation
---

- [Upgrading from 5.0.0 or later](#upgrading-to-the-latest-version-of-sensu-go-from-5-0-0-or-later)
- [Upgrading from 1.x or later](#upgrading-to-sensu-go-from-sensu-core-1-x)

## Upgrading to the latest version of Sensu Go from 5.0.0 or later

To upgrade to the latest version of Sensu Go from version 5.0.0 or later, first [install the latest packages][23].

Then restart the services.

{{< highlight shell >}}
# Restart the Sensu agent
sudo service sensu-agent restart

# Restart the Sensu backend

> sudo service sensu-agent restart

Author: ```suggestion
sudo service sensu-backend restart
```

{{< /highlight >}}

> You can verify the verson using the Sensu command line tools.

Author: ```suggestion
You can verify the version using the Sensu command line tools.
```
